base_m2m_custom_field/readme/CONTRIBUTORS.md

```diff
@@ -0,0 +1,3 @@
+- Akim Juillerat \<<akim.juillerat@camptocamp.com>\>
+- Fernando La Chica \<<fernandolachica@gmail.com>\>
```

Comment:
Why adding this contributor?

Reply:
@trisdoan can you comment or fix this? Then we can get this merged.